Gemfile.lock

```diff
@@ -844,7 +844,7 @@ CHECKSUMS
   ast (2.4.2) sha256=1e280232e6a33754cde542bc5ef85520b74db2aac73ec14acef453784447cc12
   attr_required (1.0.2) sha256=f0ebfc56b35e874f4d0ae799066dbc1f81efefe2364ca3803dc9ea6a4de6cb99
   autoprefixer-rails (10.4.16.0) sha256=40c4b14d6f26f66026cd0d4631baf18d6c56aab425b36059c8abbda17f19a706
-  avo (2.46.0) sha256=bec3d981a5e6f3e324da1aaf5379b8f6efc26f3eff5aa38c043fdb2aa1e002eb
+  avo (2.47.0) sha256=92913a585a4a2d276ea1800864053c4d989453deb0ce0ba737aeece7f6c46ded
```

Comment:
Is this intended? :thinking:

Reply:
yes i think something failed to update the checksummed version before